### PR TITLE
First draft of 1.4

### PR DESCRIPTION
**Pull request by CoralineAda**
For diffs please see version/latest/code_of_conduct.md or https://github.com/CoralineAda/contributor_covenant/tree/v1_4/diffs

### CODE REVIEW

**Comment by CoralineAda** [comment]
For diffs please see https://github.com/CoralineAda/contributor_covenant/tree/v1_4/diffs

**Comment by mprelude** [comment]
I do think 1.4 is a great improvement on 1.3, though the language is less terse.

File: README.md [hunk 1]
```diff
@@ -9,97 +9,9 @@ For an overview and usage instructions, please visit [contributor-covenant.org](
 
 ## Contributing
 
-Please feel free to submit pull requests or open issues to improve the language of this pledge.
+Please feel free to submit pull requests or open issues to improve the language
+of this document or to provide additional [translations](http://contributor-covenant.org/version/1/3/0/i18n/).
```
Comment (timdorr) [inline README.md]: Should this point to version 1.4 if that is going to end up being the released version?

**Comment by timdorr** [comment]
To make the diffing/review process easier in the future, could there be a latest.md file (or something like that) that's consistent across revisions?

File: README.md (hunk 1 above)
Comment (aredridel) [inline README.md]: Ooh, a 'current' redirector would be useful for that, actually.

Comment (mprelude) [inline README.md]: > Ooh, a 'current' redirector would be useful for that, actually.

+1 for this.

Comment (timdorr) [inline README.md]: Even better! I like that or `latest` (a common tag in the node ecosystem).

**Comment by aredridel** [comment]
I like the changes! The approach to accountability even from maintainers is a nice start. Enforcement is going to be the problem of course but it's a nice start.

**Comment by tagomoris** [comment]
Could you provide diffs by text? It's much friendly for readers, and Github supports syntax highlights for `.diff` and `.patch` file formats.
You can create it by executing `diff -u file1.md file2.md`.
https://gist.github.com/tagomoris/a9800edf8bcdcd3cb014

**Comment by CoralineAda** [comment]
@tagomoris Check the text diff inside version/latest/code_of_conduct.md

**Comment by tagomoris** [comment]
@CoralineAda Cool. Thanks.

**Pull request merged**